bridge/slack/slack.go

```diff
@@ -73,6 +73,10 @@ const (
 )
 
 func New(cfg *bridge.Config) bridge.Bridger {
+	return newBridge(cfg)
```

Comment:
Maybe we can do some leeway, add deprecation now (matterbridge 1.12.0) and remove it in 1.13.0

```golang
b := &Bslack{Config: cfg}
if !strings.HasPrefix(b.GetString(tokenConfig), "xoxb") {
      b.Log.Error("Non-bot token detected. It is STRONGLY recommended to use a proper bot-token instead.")
      b.Log.Error("Legacy tokens may be deprecated by slack at short notice. See the Matterbridge GitHub wiki for a migration guide.")
      b.Log.Error("To continue using a legacy token please move your configuration to a \"slack-legacy\" bridge instead.")
      b.Log.Error("Delaying for 30 seconds. The next release of matterbridge will exit here.")
      time.Sleep(time.Second * 30)
      return NewLegacy(cfg)
}
return newBridge(cfg)
```

Reply:
No, the delay should stay. Otherwise users won't see this message and the next version of matterbridge will break their config.

If you don't want the delay and any message, do the migration or move to `slack-legacy` as the message says:  `To continue using a legacy token please move your configuration to a "slack-legacy" bridge instead."` 

You can just edit your config-file to use "slack-legacy" instead of slack.

Reply:
Fair enough. Thank you for the code suggestion @42wim. Have taken it with a slight rewrite of the last "delay" line.